docs/azure_arc_jumpstart/azure_arc_data/microk8s/azure/arm_template/mssql_mi/_index.md

```diff
@@ -0,0 +1,212 @@
+
+  ```shell
+  az group create --name <Name of the Azure resource group> --location <Azure Region>
+  az deployment group create \
+  --resource-group <Name of the Azure resource group> \
+  --name <The name of this deployment> \
+  --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/microk8s/azure/arm_template/azuredeploy.json \
+  --parameters <The *azuredeploy.parameters.json* parameters file location>
+  ```
+
+  > **Note: Make sure that you are using the same Azure resource group name as the one you've just used in the `azuredeploy.parameters.json` file**
+
+  For example:
+
+  ```shell
+  az group create --name Arc-Data-Microk8s --location "East US"
+  az deployment group create \
+  --resource-group Arc-Data-Microk8s \
+  --name arcdatademo \
+  --template-uri https://raw.githubusercontent.com/microsoft/azure_arc/main/azure_arc_data_jumpstart/microk8s/azure/arm_template/azuredeploy.json \
+  --parameters azuredeploy.parameters.json
+  --parameters templateBaseUrl="https://raw.githubusercontent.com/your--github--handle/azure_arc/microk8s-data/azure_arc_data_jumpstart/microk8s/azure/arm_template/"
+  ```
+
+  > **Note: The deployment time for this scenario can take ~15-20min**
+
+  ![Deployment time](./02.png)
+
+- Once Azure resources has been provisioned, you will be able to see it in the Azure portal. At this point, the resource group should have **13** various Azure resources deployed.
+
+  ![ARM template deployment completed](./03.png)
+
+  ![New Azure resource group with all resources](./04.png)
+
+## Windows Login & Post Deployment
+
+- Now that first phase of the automation is completed, it is time to RDP to the Client VM using it's public IP.
+
+  ![Client VM public IP](./05.png)
+
+- At first login, as mentioned in the "Automation Flow" section above, the [_DataServicesLogonScript_](https://github.com/microsoft/azure_arc/blob/main/azure_arc_data_jumpstart/microk8s/azure/arm_template/artifacts/DataServicesLogonScript.ps1) PowerShell logon script will start it's run.
+
+- Let the script run it's course and **do not close** the PowerShell session, this will be done for you once completed.
+
+  ![PowerShell logon script run](./06.gif)
+
+  Once the script will finish it's run, the logon script PowerShell session will be closed, the Windows wallpaper will change and both the Azure Arc Data Controller and the SQL Managed Instance will be deployed on the cluster and be ready to use:
+
+  ![Wallpaper Change](./07.png)
+
+- Since this scenario is deploying the Azure Arc Data Controller and SQL Managed Instance, you will also notice additional newly deployed Azure resources in the resources group (at this point you should have **17** various Azure resources deployed). The important ones to notice are:
```

Comment:
```suggestion
- Since this scenario is deploying the Azure Arc Data Controller and SQL Managed Instance, you will also notice additional newly deployed Azure resources in the resources group (at this point you should have **17 various Azure resources deployed**. The important ones to notice are:
```